UIKitVK/UIKitVK/Controller/GroupTableViewController.swift

```diff
@@ -13,6 +13,15 @@ final class GroupTableViewController: UITableViewController {
         }
     }
 
+    private lazy var service = NetworkService()
```

Comment:
```suggestion
    private lazy var service = NetworkService()
```
зачем здесь lazy ?
 при первой загрузке мы всегда будем его дергать, так как он лежит внутри viewDidLoad
 лежал бы в методе по нажатию на кнопку был бы lazy